Support for .jsonc extension

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I wanted to add some comments to my theme file. While oh-my-posh doesn't have an issue with `//` lines in the file, my editor treats it as invalid.

When I changed the extension of my file from `.json` to `.jsonc`, oh-my-posh reported a configuration error and fell back to the default theme.

### Theme

Custom

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 18.0.1

Shell: pwsh (7.3.6)

Segments:

ConsoleTitle(true)                        -   4 ms - pwsh in OneDrive
session(true)                             -   1 ms -  username
path(true)                                -   3 ms -   OneDrive
git(false)                                -  14 ms -
root(false)                               -   0 ms -
exit(true)                                -   0 ms -  CONFIG ERROR
node(false)                               -   1 ms -
go(false)                                 -   0 ms -
python(false)                             -   1 ms -
shell(true)                               -   0 ms - in pwsh
time(true)                                -   0 ms - at 11:03:20

Run duration: 60.3909ms

Cache path: C:\Users\username\AppData\Local\oh-my-posh

Config path: C:\Users\username\OneDrive\username.omp.jsonc

Logs:

[DEBUG] 11:03:20.031 shell.go:Getenv:398 → NO DATA
[TRACE] 11:03:20.031 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 11:03:20.031 shell.go:Getenv:398 → C:\Users\username\AppData\Local
[TRACE] 11:03:20.031 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 11:03:20.032 shell_windows.go:CachePath() - 532.5µs
[DEBUG] 11:03:20.033 shell.go:Getenv:398 → C:\Users\username\OneDrive\username.omp.jsonc
[TRACE] 11:03:20.033 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 11:03:20.062 shell.go:Shell:645 → process name: pwsh.exe
[TRACE] 11:03:20.062 shell.go:Shell() - 28.502ms
[TRACE] 11:03:20.062 shell.go:resolveConfigPath() - 28.5726ms
[TRACE] 11:03:20.062 shell.go:Init() - 30.4053ms
[TRACE] 11:03:20.062 shell.go:Flags() - 0s
[ERROR] 11:03:20.062 config.go:loadConfig:154 → not register decoder for the format: jsonc
[TRACE] 11:03:20.062 config.go:loadConfig() - 522.4µs
[TRACE] 11:03:20.062 shell.go:Flags() - 0s
[DEBUG] 11:03:20.062 shell.go:Getenv:398 → NO DATA
[TRACE] 11:03:20.062 shell.go:Getenv(OMP_CACHE_DISABLED) - 62.3µs
[TRACE] 11:03:20.062 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[TRACE] 11:03:20.062 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.062 shell_windows.go:WindowsRegistryKeyValue:206 → ColorizationColor(DWORD): 0xC40078D4
[TRACE] 11:03:20.062 shell.go:Shell() - 0s
[DEBUG] 11:03:20.062 shell.go:Getenv:398 → 7.3.6
[TRACE] 11:03:20.062 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 11:03:20.062 debug.go:PrintDebug:23 → Segment: Title
[TRACE] 11:03:20.063 shell_windows.go:Root() - 0s
[TRACE] 11:03:20.063 shell.go:Shell() - 0s
[TRACE] 11:03:20.063 shell.go:StatusCodes() - 0s
[TRACE] 11:03:20.063 shell_windows.go:IsWsl() - 0s
[DEBUG] 11:03:20.063 shell.go:TemplateCache:815 ↓
    environment: [ALLUSERSPROFILE=C:\ProgramData APPDATA=C:\Users\username\AppData\Roaming CommonProgramFiles=C:\Program Files\Common Files CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files CommonProgramW6432=C:\Program Files\Common Files COMPUTERNAME=username-SB3-2 ComSpec=C:\WINDOWS\system32\cmd.exe CONDA_PROMPT_MODIFIER=False DriverData=C:\Windows\System32\Drivers\DriverData HOMEDRIVE=C: HOMEPATH=\Users\username LOCALAPPDATA=C:\Users\username\AppData\Local LOGONSERVER=\\username-SB3-2 NUMBER_OF_PROCESSORS=8 OneDrive=C:\Users\username\OneDrive - Company OneDriveCommercial=C:\Users\username\OneDrive - Company OneDriveConsumer=C:\Users\username\OneDrive OS=Windows_NT Path=C:\Program Files\PowerShell\7;C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files (x86)\gnupg\bin;C:\Program Files (x86)\Microsoft SQL Server\160\DTS\Binn\;C:\Program Files\Azure Data Studio\bin;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files\nodejs\;C:\Program Files\Docker\Docker\resources\bin;C:\Program Files\dotnet\;C:\Program Files\PowerShell\7\;C:\Program Files\Git\cmd;C:\Users\username\AppData\Local\Programs\Python\Python311\Scripts\;C:\Users\username\AppData\Local\Programs\Python\Python311\;C:\Users\username\AppData\Local\Microsoft\WindowsApps;C:\Users\username\AppData\Local\Programs\oh-my-posh\bin;C:\Users\username\AppData\Local\Programs\Bicep CLI;C:\Users\username\AppData\Local\Programs\Microsoft VS Code\bin;C:\Program Files\Azure Data Studio\bin;C:\Users\username\.dotnet\tools;C:\Users\username\AppData\Roaming\npm;C:\Users\username\AppData\Local\Keybase\ PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.CPL POSH_AZURE_ENABLED=True POSH_AZURE_SUBSCRIPTION={
      "Name": "A.C Generic Research (06) (239a1858-b106-40f0-97b8-d7fac9abbf52) - 70c53e61-5cd8-4f91-b770-5debfdd4d0ec - sven@aelterman.cloud",
      "Account": {
        "Id": "sven@aelterman.cloud",
        "Type": "User",
        "Tenants": [
          "70c53e61-5cd8-4f91-b770-5debfdd4d0ec"
        ],
        "AccessToken": null,
        "Credential": null,
        "TenantMap": {},
        "CertificateThumbprint": null,
        "ExtendedProperties": {
          "Tenants": "70c53e61-5cd8-4f91-b770-5debfdd4d0ec",
          "Subscriptions": "ee375095-d8b9-46cb-b7ae-53686660f82d,68ec4f79-589c-4b65-9916-5fe7f5d385c2,2804b8a3-d41c-4603-815c-77d8813d196e,bbdcb18c-98b9-4021-b3e7-8dc8ed798ba9,c79eee7a-38c3-45e3-940b-9de462c21bbe,05bca35e-0dfa-455a-a4eb-9f9ea72df723,74372aa9-54b9-4695-8b11-aa5d2205c0ae,a24cecd4-4761-4a48-95f0-b57daa783b80,239a1858-b106-40f0-97b8-d7fac9abbf52,d8d33e70-e25a-4d1d-a112-e9cd7d015f5c",
          "HomeAccountId": "2f439cdc-dfa9-4130-a49d-c363fe380ad1.70c53e61-5cd8-4f91-b770-5debfdd4d0ec"
        }
      },
      "Environment": {
        "Name": "AzureCloud",
        "Type": "Discovered",
        "EnableAdfsAuthentication": false,
        "OnPremise": false,
        "ActiveDirectoryServiceEndpointResourceId": "https://management.core.windows.net/",
        "AdTenant": "common",
        "GalleryUrl": null,
        "ManagementPortalUrl": "https://portal.azure.com",
        "ServiceManagementUrl": "https://management.core.windows.net/",
        "PublishSettingsFileUrl": "https://go.microsoft.com/fwlink/?LinkID=301775",
        "ResourceManagerUrl": "https://management.azure.com/",
        "SqlDatabaseDnsSuffix": ".database.windows.net",
        "StorageEndpointSuffix": "core.windows.net",
        "ActiveDirectoryAuthority": "https://login.microsoftonline.com",
        "GraphUrl": "https://graph.windows.net/",
        "GraphEndpointResourceId": "https://graph.windows.net/",
        "TrafficManagerDnsSuffix": "trafficmanager.net",
        "AzureKeyVaultDnsSuffix": "vault.azure.net",
        "DataLakeEndpointResourceId": "https://datalake.azure.net/",
        "AzureDataLakeStoreFileSystemEndpointSuffix": "azuredatalakestore.net",
        "AzureDataLakeAnalyticsCatalogAndJobEndpointSuffix": "azuredatalakeanalytics.net",
        "AzureKeyVaultServiceEndpointResourceId": "https://vault.azure.net",
        "ContainerRegistryEndpointSuffix": "azurecr.io",
        "AzureOperationalInsightsEndpointResourceId": "https://api.loganalytics.io",
        "AzureOperationalInsightsEndpoint": "https://api.loganalytics.io/v1",
        "AzureAnalysisServicesEndpointSuffix": "asazure.windows.net",
        "AnalysisServicesEndpointResourceId": "https://region.asazure.windows.net",
        "AzureAttestationServiceEndpointSuffix": "attest.azure.net",
        "AzureAttestationServiceEndpointResourceId": "https://attest.azure.net",
        "AzureSynapseAnalyticsEndpointSuffix": "dev.azuresynapse.net",
        "AzureSynapseAnalyticsEndpointResourceId": "https://dev.azuresynapse.net",
        "VersionProfiles": [],
        "ExtendedProperties": {
          "MicrosoftGraphEndpointResourceId": "https://graph.microsoft.com/",
          "MicrosoftGraphUrl": "https://graph.microsoft.com",
          "AzureAttestationServiceEndpointResourceId": "https://attest.azure.net",
          "AzureAttestationServiceEndpointSuffix": "attest.azure.net",
          "AzureSynapseAnalyticsEndpointResourceId": "https://dev.azuresynapse.net",
          "AzureSynapseAnalyticsEndpointSuffix": "dev.azuresynapse.net",
          "OperationalInsightsEndpoint": "https://api.loganalytics.io/v1",
          "OperationalInsightsEndpointResourceId": "https://api.loganalytics.io",
          "AzureAnalysisServicesEndpointSuffix": "asazure.windows.net",
          "AnalysisServicesEndpointResourceId": "https://region.asazure.windows.net",
          "ManagedHsmServiceEndpointResourceId": "https://managedhsm.azure.net",
          "ManagedHsmServiceEndpointSuffix": "managedhsm.azure.net",
          "AzurePurviewEndpointSuffix": "purview.azure.net",
          "AzurePurviewEndpointResourceId": "https://purview.azure.net",
          "AzureAppConfigurationEndpointSuffix": "azconfig.io",
          "AzureAppConfigurationEndpointResourceId": "https://azconfig.io",
          "ContainerRegistryEndpointResourceId": "https://management.azure.com"
        },
        "BatchEndpointResourceId": "https://batch.core.windows.net/"
      },
      "Subscription": {
        "Id": "239a1858-b106-40f0-97b8-d7fac9abbf52",
        "Name": "A.C Generic Research (06)",
        "State": "Enabled",
        "SubscriptionId": "239a1858-b106-40f0-97b8-d7fac9abbf52",
        "TenantId": "70c53e61-5cd8-4f91-b770-5debfdd4d0ec",
        "HomeTenantId": "70c53e61-5cd8-4f91-b770-5debfdd4d0ec",
        "ManagedByTenantIds": [
          "72f988bf-86f1-41af-91ab-2d7cd011db47"
        ],
        "CurrentStorageAccountName": null,
        "SubscriptionPolicies": {
          "LocationPlacementId": "Internal_2014-09-01",
          "QuotaId": "Internal_2014-09-01",
          "SpendingLimit": "Off"
        },
        "ExtendedProperties": {
          "HomeTenant": "70c53e61-5cd8-4f91-b770-5debfdd4d0ec",
          "ManagedByTenants": "72f988bf-86f1-41af-91ab-2d7cd011db47",
          "AuthorizationSource": "RoleBased",
          "Tenants": "70c53e61-5cd8-4f91-b770-5debfdd4d0ec",
          "SubscriptionPolices": "{\"locationPlacementId\":\"Internal_2014-09-01\",\"quotaId\":\"Internal_2014-09-01\",\"spendingLimit\":\"Off\"}",
          "Account": "sven@aelterman.cloud",
          "Environment": "AzureCloud"
        },
        "CurrentStorageAccount": null,
        "AuthorizationSource": "RoleBased",
        "Tags": null
      },
      "Tenant": {
        "Id": "70c53e61-5cd8-4f91-b770-5debfdd4d0ec",
        "TenantId": "70c53e61-5cd8-4f91-b770-5debfdd4d0ec",
        "ExtendedProperties": {},
        "TenantCategory": null,
        "Country": null,
        "CountryCode": null,
        "Name": null,
        "Domains": [],
        "DefaultDomain": null,
        "TenantType": null,
        "TenantBrandingLogoUrl": null
      },
      "TokenCache": null,
      "VersionProfile": null,
      "ExtendedProperties": {}
    } POSH_CURSOR_COLUMN=1 POSH_CURSOR_LINE=44 POSH_GIT_ENABLED=False POSH_INSTALLER=winget POSH_PID=26520 POSH_SHELL_VERSION=7.3.6 POSH_THEME=C:\Users\username\OneDrive\username.omp.jsonc POSH_THEMES_PATH=C:\Users\username\AppData\Local\Programs\oh-my-posh\themes POWERLINE_COMMAND=oh-my-posh POWERSHELL_DISTRIBUTION_CHANNEL=MSI:Windows 10 Enterprise PROCESSOR_ARCHITECTURE=AMD64 PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 126 Stepping 5, GenuineIntel PROCESSOR_LEVEL=6 PROCESSOR_REVISION=7e05 ProgramData=C:\ProgramData ProgramFiles=C:\Program Files ProgramFiles(x86)=C:\Program Files (x86) ProgramW6432=C:\Program Files PSModulePath=C:\Users\username\OneDrive - Company\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\powershell\7\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules;C:\Program Files (x86)\Microsoft Azure Information Protection\Powershell PUBLIC=C:\Users\Public SESSIONNAME=Console SystemDrive=C: SystemRoot=C:\WINDOWS TEMP=C:\Users\username\AppData\Local\Temp TMP=C:\Users\username\AppData\Local\Temp UATDATA=C:\WINDOWS\CCM\UATData\D9F8C395-CAB8-491d-B8AC-179A1FE1BE77 USERDNSDOMAIN=domain.name USERDOMAIN=REDMOND USERDOMAIN_ROAMINGPROFILE=REDMOND USERNAME=username USERPROFILE=C:\Users\username windir=C:\WINDOWS WSLENV=WT_SESSION:WT_PROFILE_ID: WT_PROFILE_ID={574e775e-4f2a-5b96-ac1e-a2962a402336} WT_SESSION=975d5e65-2498-4851-bad4-fb8cd740cdb7]
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.067 shell.go:Pwd:428 → C:\Users\username\OneDrive
[TRACE] 11:03:20.067 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.067 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.067 shell.go:User:549 → username
[TRACE] 11:03:20.067 shell.go:User() - 0s
[DEBUG] 11:03:20.067 shell.go:Host:561 → username-sb3-2
[TRACE] 11:03:20.067 shell.go:Host() - 0s
[TRACE] 11:03:20.067 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.067 shell.go:Getenv:398 → NO DATA
[TRACE] 11:03:20.067 shell.go:Getenv(SHLVL) - 0s
[TRACE] 11:03:20.067 shell.go:TemplateCache() - 4.1565ms
[TRACE] 11:03:20.067 shell.go:Flags() - 0s
[DEBUG] 11:03:20.067 block.go:Debug:280 → Segment: session
[TRACE] 11:03:20.068 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.068 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.068 shell.go:Getenv:398 → NO DATA
[TRACE] 11:03:20.068 shell.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 11:03:20.068 shell.go:Getenv:398 → NO DATA
[TRACE] 11:03:20.068 shell.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 11:03:20.068 shell.go:TemplateCache() - 0s
[TRACE] 11:03:20.068 shell.go:TemplateCache() - 0s
[TRACE] 11:03:20.068 shell.go:Shell() - 0s
[DEBUG] 11:03:20.068 block.go:Debug:280 → Segment: path
[TRACE] 11:03:20.068 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.068 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:Pwd() - 0s
[TRACE] 11:03:20.068 shell.go:Shell() - 0s
[TRACE] 11:03:20.068 shell.go:Flags() - 0s
[DEBUG] 11:03:20.068 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 11:03:20.068 properties.go:GetString:28 → 
[DEBUG] 11:03:20.068 properties.go:GetString:28 → 
[DEBUG] 11:03:20.068 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.068 properties.go:GetString:28 → ~
[DEBUG] 11:03:20.068 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 11:03:20.068 shell.go:PathSeparator() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:PathSeparator() - 0s
[TRACE] 11:03:20.068 shell.go:PathSeparator() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:PathSeparator() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:PathSeparator() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:PathSeparator() - 0s
[TRACE] 11:03:20.068 shell.go:PathSeparator() - 0s
[DEBUG] 11:03:20.068 properties.go:GetString:28 → folder
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:GOOS() - 0s
[TRACE] 11:03:20.068 shell.go:PathSeparator() - 0s
[DEBUG] 11:03:20.068 properties.go:GetString:28 → NO DATA
[TRACE] 11:03:20.068 shell.go:PathSeparator() - 0s
[DEBUG] 11:03:20.068 properties.go:GetString:28 → \
[TRACE] 11:03:20.068 shell.go:Pwd() - 0s
[TRACE] 11:03:20.068 shell_windows.go:IsWsl() - 0s
[TRACE] 11:03:20.068 shell.go:StackCount() - 0s
[DEBUG] 11:03:20.070 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 11:03:20.070 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 11:03:20.070 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 11:03:20.070 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 11:03:20.070 shell_windows.go:DirIsWritable() - 2.1287ms
[TRACE] 11:03:20.070 shell.go:TemplateCache() - 0s
[TRACE] 11:03:20.071 shell.go:TemplateCache() - 0s
[TRACE] 11:03:20.071 shell.go:Shell() - 0s
[DEBUG] 11:03:20.071 block.go:Debug:280 → Segment: git
[TRACE] 11:03:20.071 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.071 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.071 shell.go:GOOS() - 0s
[TRACE] 11:03:20.071 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.085 shell.go:CommandPath:600 → C:\Program Files\Git\cmd\git.exe
[TRACE] 11:03:20.085 shell.go:CommandPath(git.exe) - 13.445ms
[TRACE] 11:03:20.085 shell.go:HasCommand(git.exe) - 13.445ms
[TRACE] 11:03:20.085 shell.go:Pwd() - 0s
[ERROR] 11:03:20.085 shell.go:HasParentFilePath:733 → CreateFile .git: The system cannot find the file specified.
[TRACE] 11:03:20.085 shell.go:HasParentFilePath(.git) - 637.6µs
[DEBUG] 11:03:20.085 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 11:03:20.085 block.go:Debug:280 → Segment: root
[TRACE] 11:03:20.085 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.085 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.085 shell.go:GOOS() - 0s
[TRACE] 11:03:20.086 shell_windows.go:Root() - 520.5µs
[DEBUG] 11:03:20.086 block.go:Debug:280 → Segment: exit
[DEBUG] 11:03:20.086 properties.go:GetString:28 → {{ .Code }}
[TRACE] 11:03:20.086 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.086 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.086 shell.go:GOOS() - 0s
[TRACE] 11:03:20.086 shell.go:StatusCodes() - 0s
[TRACE] 11:03:20.087 shell.go:TemplateCache() - 0s
[DEBUG] 11:03:20.087 properties.go:GetBool:22 → always_enabled: true
[TRACE] 11:03:20.087 shell.go:TemplateCache() - 0s
[TRACE] 11:03:20.087 shell.go:Shell() - 0s
[TRACE] 11:03:20.087 shell.go:Flags() - 0s
[DEBUG] 11:03:20.087 block.go:Debug:280 → Segment: node
[TRACE] 11:03:20.087 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.087 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.087 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.087 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .npmrc .vue]
[DEBUG] 11:03:20.087 properties.go:GetStringArray:52 → folders: []
[DEBUG] 11:03:20.087 properties.go:GetBool:22 → home_enabled: false
[TRACE] 11:03:20.087 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.087 shell_windows.go:56 → C:\Users\username
[DEBUG] 11:03:20.087 properties.go:GetString:28 → files
[DEBUG] 11:03:20.087 properties.go:GetBool:22 → fetch_package_manager: false
[TRACE] 11:03:20.087 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.087 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.087 shell.go:HasFiles(*.js) - 684.1µs
[TRACE] 11:03:20.087 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.088 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.088 shell.go:HasFiles(*.ts) - 646.8µs
[TRACE] 11:03:20.088 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.088 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.088 shell.go:HasFiles(package.json) - 0s
[TRACE] 11:03:20.088 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.088 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.088 shell.go:HasFiles(.nvmrc) - 0s
[TRACE] 11:03:20.088 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.088 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.088 shell.go:HasFiles(pnpm-workspace.yaml) - 0s
[TRACE] 11:03:20.088 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.088 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.088 shell.go:HasFiles(.pnpmfile.cjs) - 564.8µs
[TRACE] 11:03:20.088 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.089 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.089 shell.go:HasFiles(.npmrc) - 78.5µs
[TRACE] 11:03:20.089 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.089 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.089 shell.go:HasFiles(.vue) - 0s
[DEBUG] 11:03:20.089 block.go:Debug:280 → Segment: go
[TRACE] 11:03:20.089 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.089 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.089 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.089 properties.go:GetStringArray:52 → extensions: [*.go go.mod]
[DEBUG] 11:03:20.089 properties.go:GetStringArray:52 → folders: []
[DEBUG] 11:03:20.089 properties.go:GetBool:22 → home_enabled: false
[TRACE] 11:03:20.089 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.089 shell_windows.go:56 → C:\Users\username
[DEBUG] 11:03:20.089 properties.go:GetString:28 → files
[TRACE] 11:03:20.089 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.089 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.089 shell.go:HasFiles(*.go) - 571.4µs
[TRACE] 11:03:20.089 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.089 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.089 shell.go:HasFiles(go.mod) - 0s
[DEBUG] 11:03:20.089 block.go:Debug:280 → Segment: python
[DEBUG] 11:03:20.089 properties.go:GetString:28 → files
[TRACE] 11:03:20.089 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.089 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.089 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.089 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 11:03:20.089 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv env venv-win pyenv-win]
[DEBUG] 11:03:20.089 properties.go:GetBool:22 → home_enabled: false
[TRACE] 11:03:20.089 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.089 shell_windows.go:56 → C:\Users\username
[DEBUG] 11:03:20.089 properties.go:GetBool:22 → fetch_virtual_env: false
[TRACE] 11:03:20.089 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.089 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.089 shell.go:HasFiles(*.py) - 0s
[TRACE] 11:03:20.089 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.090 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.090 shell.go:HasFiles(*.ipynb) - 1.251ms
[TRACE] 11:03:20.090 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.091 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.091 shell.go:HasFiles(pyproject.toml) - 121.9µs
[TRACE] 11:03:20.091 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.091 shell.go:HasFiles:450 → false
[TRACE] 11:03:20.091 shell.go:HasFiles(venv.bak) - 0s
[DEBUG] 11:03:20.091 shell.go:HasFolder:493 → false
[TRACE] 11:03:20.091 shell.go:HasFolder(.venv) - 0s
[DEBUG] 11:03:20.091 shell.go:HasFolder:493 → false
[TRACE] 11:03:20.091 shell.go:HasFolder(venv) - 0s
[DEBUG] 11:03:20.091 shell.go:HasFolder:493 → false
[TRACE] 11:03:20.091 shell.go:HasFolder(virtualenv) - 0s
[DEBUG] 11:03:20.091 shell.go:HasFolder:493 → false
[TRACE] 11:03:20.091 shell.go:HasFolder(env) - 0s
[DEBUG] 11:03:20.091 shell.go:HasFolder:493 → false
[TRACE] 11:03:20.091 shell.go:HasFolder(venv-win) - 0s
[DEBUG] 11:03:20.091 shell.go:HasFolder:493 → false
[TRACE] 11:03:20.091 shell.go:HasFolder(pyenv-win) - 0s
[DEBUG] 11:03:20.091 block.go:Debug:280 → Segment: shell
[TRACE] 11:03:20.091 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.091 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.091 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.091 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[TRACE] 11:03:20.091 shell.go:Shell() - 0s
[TRACE] 11:03:20.091 shell.go:Flags() - 0s
[TRACE] 11:03:20.091 shell.go:TemplateCache() - 0s
[TRACE] 11:03:20.091 shell.go:TemplateCache() - 0s
[TRACE] 11:03:20.091 shell.go:Shell() - 0s
[DEBUG] 11:03:20.091 block.go:Debug:280 → Segment: time
[TRACE] 11:03:20.091 shell.go:Pwd() - 0s
[DEBUG] 11:03:20.091 shell_windows.go:56 → C:\Users\username
[TRACE] 11:03:20.091 shell.go:GOOS() - 0s
[DEBUG] 11:03:20.091 properties.go:GetString:28 → 15:04:05
[TRACE] 11:03:20.091 shell.go:TemplateCache() - 0s
[TRACE] 11:03:20.092 shell.go:TemplateCache() - 0s
[TRACE] 11:03:20.092 shell.go:Shell() - 0s
[DEBUG] 11:03:20.092 shell.go:Getenv:398 → C:\Users\username\AppData\Local
[TRACE] 11:03:20.092 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 11:03:20.092 shell_windows.go:CachePath() - 0s
[TRACE] 11:03:20.092 shell.go:Flags() - 0s
```

--- a/src/engine/config.go
+++ b/src/engine/config.go
@@ -133,12 +133,17 @@ func loadConfig(env platform.Environment) *Config {
 	cfg.origin = configFile
 	cfg.Format = strings.TrimPrefix(filepath.Ext(configFile), ".")
 	cfg.env = env
-	if cfg.Format == "yml" {
+
+	// support different extensions
+	switch cfg.Format {
+	case "yml":
 		cfg.Format = YAML
+	case "jsonc":
+		cfg.Format = JSON
 	}
 
-	config.AddDriver(yaml.Driver)
-	config.AddDriver(json.Driver)
+	config.AddDriver(yaml.Driver.WithAliases("yaml", "yml"))
+	config.AddDriver(json.Driver.WithAliases("json", "jsonc"))
 	config.AddDriver(toml.Driver)
 
 	if config.Default().IsEmpty() {

--- a/src/platform/shell.go
+++ b/src/platform/shell.go
@@ -632,9 +632,10 @@ func (env *Shell) Flags() *Flags {
 
 func (env *Shell) Shell() string {
 	defer env.Trace(time.Now())
-	if env.CmdFlags.Shell != "" {
+	if len(env.CmdFlags.Shell) != 0 {
 		return env.CmdFlags.Shell
 	}
+	env.Debug("no shell name provided in flags, trying to detect it")
 	pid := os.Getppid()
 	p, _ := process.NewProcess(int32(pid))
 	name, err := p.Name()

